### ISSUE TITLE
[Feature Request] request  to support a easier way to custom multiple-selected-table row 's hightlight bg color

### ISSUE THREAD

**Issue by xiangqunXiao** [element-plus/element-plus#5197]
### Existing Component
Yes

### Component Name
table

### Description
Currently, I need to do a lot of work when using `row-class-name` to custom bg-color of selected row, which need to handle loop logic code with select event.
If we add an className with css var for selection row in source code, developers are able to control selected row bg-color by change css var value. Is it a better way?

**Comment by YunYouJun** [comment]
Of course. If you just want to change the color, css var is always more elegant.
